docs/testing/testcafe_functional_testing.md

```diff
@@ -0,0 +1,150 @@
+---
+id: testcafe_functional_testing
+title: Functional Testing with TestCafe
+sidebar_label: TestCafe - Functional Testing
+---
+
+## TestCafe: Functional tests
+
```

Comment:
Add a section here about what TestCafe is, what it is generally used for and why it is beneficial as well as a link to TestCafe itself.